scripts/verilator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#-------------------------------------
+# Installing verilator for cocotb
+#-------------------------------------
+
+# REQUIRED verilator version
+export VL_VERSION=v4.106
```

Comment:
This I understand but I wonder if you think it would also be better if we also put the actual steps to install it? 

Gui made this script and I thought it was useful so I thought it would be great. It's similar to the Verilator steps but without the other clutter. The CI on the other hand, I could follow but it's like more manual to run. I mean it isn't a shell script anyway.